src/backend/database/migrations/tenant/2025_01_18_100835_drop_user_table.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up() {
+        // Check if the users table exists in the tenant connection
+        if (Schema::connection('tenant')->hasTable('users')) {
```

Comment:
Nit: I don't think we need this. If `tenant.users` doesn't exist at the time the migration is run, then something is fundamentally wrong and it's ok if application of the migration fails.